issue_15/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js" defer></script>
+    <title>もりけん塾 JS課題15</title>
+</head>
+
+<body>
+    <div class="btn__wrapper show-btn__wrapper">
+        <button type="button" class="btn show-btn" id="js-show-btn">モーダルを表示する</button>
+    </div>
+    <div class="modal" id="js-modal">
+        <form action="" method="post" id="js-form">
```

Comment:
IMO
If it is blank, you may want to delete it.
```suggestion
        <form method="post" id="js-form">
```

Reply:
I did this because I saw that it is better to use a blank field for security reasons.
However, I found out that this is a case where form values are managed by PHP, etc., and it did not seem to be relevant in this case, so I removed it.
Thank you for pointing this out:raised_hands:

The omission of the name attribute value has also been corrected.
d5bc2d8a59b2ec3c1ea3b2a59e545b77db969d52

Reply:
Thank you for the heads up! I looked into it myself and it made sense to leave the action attribute blank, I didn't study it enough. If it is blank, it is optional to omit it unless there is a special reason, so I did not see a problem with erasing it. Thank you for the fix!